root/cgi-bin/print_ok.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import os
+
+def print_ok():
+	print(f"Content-type: text/plain\n\n")
+	print(f"OK")
```

Comment:
細かいのですが、`f` は変数を文字列に埋め込む時に使うので、今回はいらなさそうです
```suggestion
def print_ok():
	print("Content-type: text/plain\n\n")
	print("OK")
```